crates/red_knot_python_semantic/resources/mdtest/type_properties/tuples_containing_never.md

```diff
@@ -19,6 +19,11 @@ static_assert(is_equivalent_to(Never, tuple[int, Never]))
 static_assert(is_equivalent_to(Never, tuple[int, Never, str]))
 static_assert(is_equivalent_to(Never, tuple[int, tuple[str, Never]]))
 static_assert(is_equivalent_to(Never, tuple[tuple[str, Never], int]))
+
+def _(x: tuple[Never], y: tuple[int, Never], z: tuple[Never, int]):
+    reveal_type(x)  # revealed: Never
+    reveal_type(y)  # revealed: Never
+    reveal_type(z)  # revealed: Never
```

Comment:
Not that it's particularly important, but do these really test something other than the assertions in lines 16-18?

Reply:
I see your point, but IMO it "says the same thing in a different way", and I think that can be valuable for human readers of the test

Reply:
To add to that, I'd say it's testing the "simplified to `Never`" part specifically and more precisely.